src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * added support for variadics in named arguments
  * added PSR-11 `ContainerBagInterface` and its `ContainerBag` implementation to access parameters as-a-service
+ * The `container.service_locator` tag of `ServiceLocator`s is now autoconfigured.
```

Comment:
This should go into the ChangeLog of FrameworkBundle, bot the DI component.

Reply:
Ah, you are correct! I was confused by the path of the `FrameworkExtension`. Is fixed.